lib/person.rb

```diff
@@ -1,8 +1,13 @@
-class Person
+require_relative './nameable'
+require_relative './capitalize_decorator'
+require_relative './trimmer_decorator'
+
```

Comment:
- [ ] Kindly remove these statements here. When I try to create a Person object in other files to test your code, I get errors due to these lines of code. Remember that the only class you need to import in the `person.rb` file is the `Nameable` class. Kindly implement this change. 👍

Reply:
@Gambit142 Sure, I tested the functionality and forgot to remove those, thank you for your feedback!